flink-table-store-core/src/test/java/org/apache/flink/table/store/file/operation/FileStoreCommitTest.java

```diff
@@ -150,9 +150,12 @@ public void testOverwritePartialCommit() throws Exception {
 
         TestFileStore store = createStore(false);
         store.commitData(
-                data1.values().stream().flatMap(Collection::stream).collect(Collectors.toList()),
-                gen::getPartition,
-                kv -> 0);
+                        data1.values().stream()
+                                .flatMap(Collection::stream)
+                                .collect(Collectors.toList()),
+                        gen::getPartition,
+                        kv -> 0)
+                .forEach(snapshot -> assertThat(snapshot.getLogOffsets().get(0)).isEqualTo(1));
```

Comment:
I didn't quite understand your test. As I understand it, the test logic should be:
- commit with full log offsets, assert.
- commit with partial log offsets (Complemented by the previous snapshot), assert.